### ISSUE TITLE
Test Hardline Network Connections

### ISSUE THREAD

**Issue by Don-van-Houts** [hoobs-org/hoobsd#41]
I have a question,

I had running, the latest version off HOOBS was running on my Raspberry PI B.
Now I have installed HOOBS 4 Beta on the SD card and plugged it in my Raspberry, but the problem now is that I can’t get it running if I go too          “ http://hoobs.local” .
Can you please tell me what I am doing wrong or am I forgetting something?

I hope to hear from you soon.

Met vriendelijke groet/ greetings

Don van Houts

**Comment by Don-van-Houts** [comment]
I have tested the connection by switching the Ethernet cable’s and still encounter the same issue. So the connection on the router as well as the cable itself is working how it should be.
I think the problem is in the Raspberry/software.

Met vriendelijke groet,

Don van Houts

Op 17 apr. 2021 om 22:21 heeft Michael Kellsy ***@***.***> het volgende geschreven:

﻿

Reopened #41<https://github.com/hoobs-org/hoobsd/issues/41>.

**Comment by mkellsy** [comment]
The "test" is for me :)

I need to test the hardline connection and adjust the services to address this.

Sorry for the confusion.

**Comment by themick4u** [comment]
Can you login to your router and see which dhcp IP address your new instance is running on? I had to add my 4 beta to the dhcp static list for the new ip mine got.

**Comment by mkellsy** [comment]
I think it has to do with the systemd service order when we build the image.
I think this is a bug.

You can SSH into the device and see if the hoobsd service is running.
```
sudo systemctl status hoobsd.service
```

If it isn't running
```
sudo systemctl status hoobsd.service
```

However this will not start back up if you reboot the Pi, which is why I have to review our build process.

**Comment by Don-van-Houts** [comment]
I am trying to SSH into the device;
Can you tell me what the password is?
Met vriendelijke groet,[cid:796D5700-45F8-4E7D-B829-1D4ED7B27B02]


Don van Houts





Op 18 apr. 2021, om 22:54 heeft Michael Kellsy ***@***.******@***.***>> het volgende geschreven:



I think it has to do with the systemd service order when we build the image.
I think this is a bug.

You can SSH into the device and see if the hoobsd service is running.

sudo systemctl status hoobsd.service


If it isn't running

sudo systemctl status hoobsd.service


However this will not start back up if you reboot the Pi, which is why I have to review our build process.

**Comment by mkellsy** [comment]
username: hoobs
password: hoobsadmin

**Comment by Don-van-Houts** [comment]
Can you tell me why the sudo codes are exactly the same?
Or is the code for checking a running Hoobs the same as for a none running Hoobs?

Or is it a mistake and did you type the same code by accident?


Met vriendelijke groet,

Don van Houts





Op 18 apr. 2021, om 22:54 heeft Michael Kellsy ***@***.******@***.***>> het volgende geschreven:



I think it has to do with the systemd service order when we build the image.
I think this is a bug.

You can SSH into the device and see if the hoobsd service is running.

sudo systemctl status hoobsd.service


If it isn't running

sudo systemctl status hoobsd.service


However this will not start back up if you reboot the Pi, which is why I have to review our build process.

**Comment by Don-van-Houts** [comment]
I tried the given password below but this isn’t working


Met vriendelijke groet,

Don van Houts





Op 19 apr. 2021, om 03:12 heeft Michael Kellsy ***@***.******@***.***>> het volgende geschreven:



username: hoobs
password: hoobsadmin

**Comment by mkellsy** [comment]
The sudo password is simply hoobsadmin.

I am assuming that you are using our image. The username and password are set in our build process.

Here's the SSH command
```
ssh hoobs@hoobs.local
```

Password is hoobsadmin.

If you want to use HelM (our web based terminal)
Go to this address http://hoobs.local:9090

Username: hoobs
Password: hoobsadmin

**Comment by mkellsy** [comment]
To continue this issue. (This is for me)

I updated balina WiFi-Connect to version 4.4.5

NEXT STEP
Build a dev version of the image and test it with a hardline connection

**Comment by mkellsy** [comment]
I verified that hardline networking works well in wifi-connect 4.4.5.

I am marking this fixed in our Image version 4.0.19.

**Comment by Don-van-Houts** [comment]
Ok. Nice.

I have bought a new Raspberry Pi 4.
And installed HOOBS 4 Beta. Works (with Ethernet cable) perfect!

If I have issues either plug-ins can I discuss this with you also or do I have to give the feedback somewhere else?

I have already have a question. If I find a plug-in and I am on the download page in HOOBS of coursei can go to the plug-in page on Git Hub. But can you tell me how I can go back (a page) to HOOBS?

I’m using a Apple MacBook Air and the HOOBS desktop app.

Thx.

Met vriendelijke groet,

Don van Houts

Op 22 apr. 2021 om 00:46 heeft Michael Kellsy ***@***.***> het volgende geschreven:

﻿

Closed #41<https://github.com/hoobs-org/hoobsd/issues/41>.

**Comment by Don-van-Houts** [comment]
I just updated to the latest version, but now the UI and the desktop app crashed.
I can not get into HOOBS anymore, not by the desktop app and not by browser.

Can you help me?


Met vriendelijke groet,

Don van Houts





Op 22 apr. 2021, om 00:46 heeft Michael Kellsy ***@***.******@***.***>> het volgende geschreven:



Closed #41<https://github.com/hoobs-org/hoobsd/issues/41>.
